db/migrate/20150226164000_add_preference_to_student.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferenceToStudent < ActiveRecord::Migration
```

Comment:
We don't need this field.

We will have one `preference` field for mentor and 3 preference fields (as defined by you after this) for the student. 

So, please remove this migration and rebuild the schema and push to this branch.